### ISSUE TITLE
Is the latest version of angular-highcharts supporting highcharts version 6.1?

### ISSUE THREAD

**Issue by haerulmutakin** [cebor/angular-highcharts#236]
Hi,
Is the latest version of angular-highcharts supporting highcharts version 6.1? 
Iam using angular-highcharts 7.0.2 (latest) and try to follow this tutorial  https://www.highcharts.com/blog/news/live-data-configuration/ .
But, i got this error.
"message": "Argument of type '{ chart: { type: string; height: number; }; title: { text: string; }; legend: { enabled: false; }; subtitle: { text: string; }; data: { csvURL: string; enablePolling: boolean; dataRefreshRate: number; }; plotOptions: { ...; }; yAxis: { ...; }; }' is not assignable to parameter of type 'Options'.\n  Types of property 'data' are incompatible.\n    Type '{ csvURL: string; enablePolling: boolean; dataRefreshRate: number; }' is not assignable to type 'DataOptions'.\n      Object literal may only specify known properties, and 'csvURL' does not exist in type 'DataOptions'.",
	"source": "ts",
	"startLineNumber": 35,
	"startColumn": 9,
	"endLineNumber": 35,
	"endColumn": 68

**csvURL**, **enablePolling** and **dataRefreshRate** are not recognized by angular-highcarts. can anyone help?

**Comment by SayakMukhopadhyay** [comment]
It is an issue with highcharts typings. I face the same issue in some modules. The only way out is to assign `any` to the variables causing an issue.

**Comment by haerulmutakin** [comment]
@SayakMukhopadhyay can you give me an example please. because it's important for me

**Comment by SayakMukhopadhyay** [comment]
Say, you are typing the `options` and `chart` like this
![image](https://user-images.githubusercontent.com/13748882/47935015-0d8a6300-deff-11e8-819d-88ede70e098c.png)
But you will see that I am getting typing error
![image](https://user-images.githubusercontent.com/13748882/47935061-2dba2200-deff-11e8-93d3-8822e4a76316.png)
So, instead I will set my `options` as type `any`.
![image](https://user-images.githubusercontent.com/13748882/47935097-44f90f80-deff-11e8-966c-35059749da4b.png)
But keep the `chart` as the correct type since it is not problematic.

Basically if you have trouble with typescript and typings, switch the type to any. You will lose the goodness of typescript but at least your code will compile.

**Comment by aviral282** [comment]
I think they have changed the way to define these properties  under :
[here](https://github.com/chartjs/Chart.js/pull/6249/files/64f27952b206982f06201b67aefbc30dc1d44c2a)
Which suggests to define the properties under dataset
